docs/Get started/why-fleet.md

```diff
@@ -31,42 +31,38 @@ Here are the platforms Fleet currently supports:
 - Linux-based IoT devices
 
 ## Lighter than air
-Fleet is lightweight and modular.  You can use it for security without using it for MDM, and vice versa.  You can turn off features you are not using.
+Fleet is lightweight and modular.  You can use it for security without MDM and vice versa.  You can turn off features you are not using.
```

Comment:
disagree with this line change.  MDM is a capability, and this change makes it less clear

Reply:
Ok, that makes sense.